services/export/service/src/main/java/org/collectionspace/services/export/CsvExportWriter.java

```diff
@@ -35,7 +41,7 @@ public class CsvExportWriter extends AbstractExportWriter {
 	private static final String AUTHORITY_SUFFIX = "AuthorityVocabulary";
 
 	private CSVPrinter csvPrinter;
-	private Map<String, Map<String, Set<String>>> refFieldsByDocType = new HashMap<>();
+	private Map<String, Table<String, String, Set<String>>> refFieldsByDocType = new HashMap<>();
```

Comment:
Storing the doc type seems unnecessary but I wasn't sure if there could ever be a collision for partNames. It seems unlikely as the export only works on a single docType at a time (though core fields can be exported as well). Maybe it would be worth it though to change.